src/annotator/config/index.js

```diff
@@ -76,75 +83,104 @@ function configurationKeys(appContext) {
  */
 const configDefinitions = {
   annotations: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.annotations,
   },
   appType: {
-    getValue: settings =>
-      settings.hostPageSetting('appType', {
-        allowInBrowserExt: true,
-      }),
+    allowInBrowserExt: true,
+    defaultValue: null,
+    getValue: settings => settings.hostPageSetting('appType'),
   },
   branding: {
+    defaultValue: null,
+    allowInBrowserExt: false,
     getValue: settings => settings.hostPageSetting('branding'),
   },
   // URL of the client's boot script. Used when injecting the client into
   // child iframes.
   clientUrl: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.clientUrl,
   },
   enableExperimentalNewNoteButton: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings =>
       settings.hostPageSetting('enableExperimentalNewNoteButton'),
   },
   group: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.group,
   },
   focus: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('focus'),
   },
   theme: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('theme'),
   },
   usernameUrl: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('usernameUrl'),
   },
   onLayoutChange: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('onLayoutChange'),
   },
   openSidebar: {
+    allowInBrowserExt: true,
+    defaultValue: false,
     coerce: toBoolean,
-    getValue: settings =>
-      settings.hostPageSetting('openSidebar', {
-        allowInBrowserExt: true,
-      }),
+    getValue: settings => settings.hostPageSetting('openSidebar'),
   },
   query: {
+    allowInBrowserExt: true,
+    defaultValue: null,
     getValue: settings => settings.query,
   },
   requestConfigFromFrame: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('requestConfigFromFrame'),
   },
   services: {
+    allowInBrowserExt: false,
+    defaultValue: null,
     getValue: settings => settings.hostPageSetting('services'),
   },
   showHighlights: {
+    allowInBrowserExt: false,
+    defaultValue: null,
```

Comment:
The default value here should be `always`. This is resolved in https://github.com/hypothesis/client/pull/3500.

Reply:
I'll leave this for now since these will land all at the same time.